ai-ml/mix-train-and-inference/kueue/patch.yaml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2024 Google LLC
+# Copyright 2024-2025 Google LLC
```

Comment:
Heads-up for future PRs: Updating the license to include the most recent year (that the file was updated) isn't necessary. No action needed. Related guidance at this Google-internal link: [go/copyright#copyright-notice](http://go/copyright#copyright-notice)